src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -622,6 +575,36 @@ protected internal bool IsArc()
             return false;
         }
 
+        /*
```

Comment:
For PowerShell project we require `/// <summary>` comment blocks, not `/*`.  Not sure if you need to follow the same convention here.